spd/models/components.py

```diff
@@ -92,41 +92,14 @@ def __init__(self, C: int, v_dim: int, u_dim: int):
         self.C = C
         self.V = nn.Parameter(torch.empty(v_dim, C))
         self.U = nn.Parameter(torch.empty(C, u_dim))
+        init_param_(self.V, fan_val=v_dim, nonlinearity="linear")
+        init_param_(self.U, fan_val=C, nonlinearity="linear")
```

Comment:
Just to confirm, the fan_val corresponds to the input dimension? Data goes through V first. Resolve comment if agree.